### DISCUSSION TITLE
Command Line Support

### DISCUSSION THREAD

**Discussion by markpahulje** [GyulyVGC/sniffnet#625] [Ideas]
Hello, could we get command line support to start the monitoring and choosing the adapter by number. IE sniffnet.exe -adapter 3 /start

**Comment by GyulyVGC** [comment]
Hi @markpahulje, what would you need this for?

This is to better understand if you only need to start the monitoring from the specified adapter or you also expect something else to be available via CLI.

**Reply by markpahulje** [reply]
Hi GyulyVGC, I just wanted to monitor the activity of a particular connection when my network went down. I set up an event trigger to launch sniffnet.exe when network went down. Likewise, you could do the same, in your program, send a warning to user either via pop-up or sound to warn connection is down in addition to writing to a disconnection log. I am trying to monitor my connect from dropped video calls over 4hrs. Hope that explains my use case. Thanks

**Reply by GyulyVGC** [reply]
Hi @markpahulje now it's possible to launch the app with a network adapter name. See #643. This will be release in v1.3.2.

**Comment by GyulyVGC** [comment]
Now this discussion has a dedicated issue. See #636.
